src/slider/slick/dots.jsx

```diff
@@ -66,18 +81,25 @@ export default class Dots extends React.Component {
             }
 
             children.push(
-                <li key={i} className={itemClazz} {...handleProp}>
-                    {
-                        (dotsRender instanceof Function) ?
-                            <span>
-                                {dotsRender(docIndex, currentSlideIndex)}
-                            </span> :
-                            <button />
-                    }
+                <li
+                    key={i}
+                    className={itemClazz}
+                    role="listitem"
```

Comment:
`role=listitem` is implicit for `<li>` with a `<ul>` or `<ol>` parent. Its not recommended to have it.